packages/fxa-auth-server/lib/senders/templates/verifyShortCode.html

```diff
@@ -11,7 +11,7 @@ <h1 style="font-family: sans-serif; font-size: 21px; line-height: 29px; font-wei
 <tr style="page-break-before: always">
   <td valign="top">
     <p style="font-family: sans-serif; font-size: 14px; line-height: 21px; font-weight: normal; margin: 0 0 21px 0; text-align: center;">
-      {{t "If yes, use this verification code:" }}
+      {{t "If yes, use this verification-code in your registration form." }}
```

Comment:
@rukshan99 Thanks for your PR. Will you update this copy to:

"If yes, use this verification code in your registration form:"

I don't think we want the dash between "verification" and "code" and we want to leave the colon instead of a period.

Reply:
Sure!

Reply:
@LZoog Hope now it's okay.

Reply:
@rukshan99 Yes, it looks great! Sorry for not mentioning this before, but will you rebase your branch on top of `main` and force push? You may know how already but to make it easy, you can run `git rebase -i main` and "fix up" your other commits (`f` on commits after the first one) into a single commit. It looks to me like the Circle CI test failures are false positives, so once you've rebased and force pushed, CI will rerun and we should be good to go!

P.S. Are you an Outreachy applicant? 🙂

Reply:
@LZoog I think now it's okay. I'm still new to contributing, so thank you for guiding me :)

No, I'm not an Outreachy applicant.

Reply:
@rukshan99 When you rebase against `main` and you "fix up" your commits into one, there should only be one single commit showing in your PR into `main`.

Something went a little funky since it's showing dependabot commits... You'll want to rebase again against latest `main`. You can discard these other commits. When you run `git log`, you should only show one commit at the top, and the 2nd to top should be the latest commit into `main`. If this is giving you a lot of trouble, you can try resetting your branch to latest main and then running `git cherry-pick f4a15185c4eed54d53c4836bde744e672735c9f1` to apply your latest commit on top of `main`.